### PR TITLE
Rtb update

### PR DESCRIPTION
**Pull request by TomLayson**

### CODE REVIEW

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 1]
```diff
@@ -54,6 +54,11 @@ The following sections list the components that make network connections to Micr
 
 The following table lists management options for each setting, beginning with Windows 10 Enterprise version 1607.
 
+ >[!IMPORTANT]
+>**If you need assistance with troubleshooting issues, please refer to:**</br> 
+> - [Keep your device running smoothly](https://support.microsoft.com/en-us/topic/keep-your-device-running-smoothly-with-recommended-troubleshooting-ec76fe10-4ac8-ce9d-49c6-757770fe68f1)</br>
+> - [CSP - Troubleshooting](https://docs.microsoft.com/en-us/windows/client-management/mdm/policy-csp-troubleshooting)
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:60]:
```suggestion
 > [!IMPORTANT]
> **If you need assistance with troubleshooting issues, please refer to:**</br> 
>  - [Keep your device running smoothly](https://support.microsoft.com/topic/keep-your-device-running-smoothly-with-recommended-troubleshooting-ec76fe10-4ac8-ce9d-49c6-757770fe68f1)</br>
> - [CSP - Troubleshooting](/windows/client-management/mdm/policy-csp-troubleshooting)
```

File: windows/privacy/manage-windows-1709-endpoints.md [hunk 2]
```diff
@@ -150,6 +150,10 @@ If you [turn off traffic for this endpoint](manage-connections-from-windows-oper
 
 ## Certificates
 
+Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA), are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-1709-endpoints.md:153]:
```suggestion
Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA) are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```

File: windows/privacy/manage-windows-1803-endpoints.md [hunk 3]
```diff
@@ -151,6 +151,10 @@ If you [turn off traffic for this endpoint](manage-connections-from-windows-oper
 
 ## Certificates
 
+Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA), are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-1803-endpoints.md:154]:
```suggestion
Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA) are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```

File: windows/privacy/manage-windows-1809-endpoints.md [hunk 4]
```diff
@@ -162,6 +162,10 @@ If you [turn off traffic for this endpoint](manage-connections-from-windows-oper
 
 ## Certificates
 
+Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA), are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-1809-endpoints.md:165]:
```suggestion
Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA) are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```

File: windows/privacy/manage-windows-1903-endpoints.md [hunk 5]
```diff
@@ -67,7 +67,9 @@ The following methodology was used to derive these network endpoints:
...
+|Certificates|Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA), are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-1903-endpoints.md:70]:
```suggestion
|Certificates|Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA) are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```

File: windows/privacy/manage-windows-1909-endpoints.md [hunk 6]
```diff
@@ -54,7 +54,9 @@ The following methodology was used to derive these network endpoints:
...
+|Certificates|Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA), are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-1909-endpoints.md:57]:
```suggestion
|Certificates|Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA) are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```

File: windows/privacy/manage-windows-2004-endpoints.md [hunk 7]
```diff
@@ -54,7 +54,9 @@ The following methodology was used to derive these network endpoints:
...
+|Certificates|Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA), are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-2004-endpoints.md:57]:
```suggestion
|Certificates|Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA) are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```

File: windows/privacy/manage-windows-20H2-endpoints.md [hunk 8]
```diff
@@ -54,7 +54,8 @@ The following methodology was used to derive these network endpoints:
...
+|Certificates|Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA), are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-20H2-endpoints.md:57]:
```suggestion
|Certificates|Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA) are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```

File: windows/privacy/manage-windows-1903-endpoints.md (hunk 5 above)
Comment (TomLayson) [reply windows/privacy/manage-windows-1903-endpoints.md:70]: The text for this item was supplied by CELA and contains specific language needed.

File: windows/privacy/manage-windows-1909-endpoints.md (hunk 6 above)
Comment (TomLayson) [reply windows/privacy/manage-windows-1909-endpoints.md:57]: Same as above.  The text for this item was supplied by CELA and contains specific language needed.

File: windows/privacy/manage-windows-21H1-endpoints.md [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+- Connecting to Microsoft Office and Windows sites to download the latest app and security updates.
+- Connecting to email servers to send and receive email.
+- Connecting to the web for every day web browsing.
+- Connecting to the cloud to store and access backups.
+- Using your location to show a weather forecast.
+
+Details about the different ways to control traffic to these endpoints are covered in [Manage connections from Windows operating system components to Microsoft services](manage-connections-from-windows-operating-system-components-to-microsoft-services.md).
+Where applicable, each endpoint covered in this topic includes a link to the specific details on how to control that traffic.
+
+The following methodology was used to derive these network endpoints:
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-21H1-endpoints.md:35]:
```suggestion
The following methodology was used to derive at these network endpoints:
```

File: windows/privacy/manage-windows-1909-endpoints.md (hunk 6 above)
Comment (TomLayson) [reply windows/privacy/manage-windows-1909-endpoints.md:57]: misread.  please disregard

File: windows/privacy/manage-windows-21H1-endpoints.md [hunk 10]
```diff
@@ -0,0 +1,162 @@
...
+
+> [!NOTE]
+> Microsoft uses global load balancers that can appear in network trace-routes. For example, an endpoint for *.akadns.net might be used to load balance requests to an Azure datacenter, which can change over time.
+
+## Windows 10 21H1 Enterprise connection endpoints
+
+|Area|Description|Protocol|Destination|
+|----------------|----------|----------|------------|
+|Apps|||[Learn how to turn off traffic to the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#bkmk-windowsstore)|
+||The following endpoint is used for the Weather app. To turn off traffic for this endpoint, either uninstall the Weather app or disable the Microsoft Store. If you disable the Microsoft store, other Store apps cannot be installed or updated. Additionally, the Microsoft Store won't be able to revoke malicious Store apps and users will still be able to open them.|HTTP|tile-service.weather.microsoft.com|
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-21H1-endpoints.md:54]:
```suggestion
||The following endpoint is used for the Weather app. To turn off traffic for this endpoint, either uninstall the Weather app or disable the Microsoft Store. If you disable the Microsoft Store, other Store apps cannot be installed or updated. Additionally, the Microsoft Store won't be able to revoke malicious Store apps and users will still be able to open them.|HTTP|tile-service.weather.microsoft.com|
```

File: windows/privacy/manage-windows-21H1-endpoints.md [hunk 11]
```diff
@@ -0,0 +1,162 @@
...
+||The following endpoint is used for OneNote Live Tile. To turn off traffic for this endpoint, either uninstall OneNote or disable the Microsoft Store. If you disable the Microsoft store, other Store apps cannot be installed or updated. Additionally, the Microsoft Store won't be able to revoke malicious Store apps and users will still be able to open them.|TLSv1.2/HTTPS/HTTP|cdn.onenote.net|
+||The following endpoint is used by the Photos app to download configuration files, and to connect to the Office 365 portal's shared infrastructure, including Office in a browser. To turn off traffic for this endpoint, either uninstall the Photos app or disable the Microsoft Store. If you disable the Microsoft store, other Store apps cannot be installed or updated. Additionally, the Microsoft Store won't be able to revoke malicious Store apps and users will still be able to open them.|TLSv1.2/HTTPS|evoke-windowsservices-tas.msedge.net
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-21H1-endpoints.md:56]:
```suggestion
||The following endpoint is used for OneNote Live Tile. To turn off traffic for this endpoint, either uninstall OneNote or disable the Microsoft Store. If you disable the Microsoft Store, other Store apps cannot be installed or updated. Additionally, the Microsoft Store won't be able to revoke malicious Store apps and users will still be able to open them.|TLSv1.2/HTTPS/HTTP|cdn.onenote.net|
||The following endpoint is used by the Photos app to download configuration files, and to connect to the Office 365 portal's shared infrastructure, including Office in a browser. To turn off traffic for this endpoint, either uninstall the Photos app or disable the Microsoft Store. If you disable the Microsoft Store, other Store apps cannot be installed or updated. Additionally, the Microsoft Store won't be able to revoke malicious Store apps and users will still be able to open them.|TLSv1.2/HTTPS|evoke-windowsservices-tas.msedge.net
```

File: windows/privacy/manage-windows-21H1-endpoints.md [hunk 12]
```diff
@@ -0,0 +1,162 @@
...
+|Certificates|Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA), are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-21H1-endpoints.md:57]:
```suggestion
|Certificates|Certificates are digital files, stored on client devices, used to both encrypt data and verify the identity of an individual or organization. Trusted root certificates issued by a certification authority (CA) are stored in a certificate trust list (CTL). The Automatic Root Certificates Update mechanism contacts Windows Updates to update the CTL. If a new version of the CTL is identified, the list of trusted root certificates cached on the local device will be updated. Untrusted certificates are certificates where the server certificate issuer is unknown or is not trusted by the service. Untrusted certificates are also stored in a list on the local device and updated by the Automatic Root Certificates Update mechanism.
```

File: windows/privacy/manage-windows-21H1-endpoints.md [hunk 13]
```diff
@@ -0,0 +1,162 @@
...
+|||HTTP|dmd.metaservices.microsoft.com|
+|Diagnostic Data|The following endpoints are used by the Connected User Experiences and Telemetry component and connects to the Microsoft Data Management service. If you turn off traffic for this endpoint, diagnostic and usage information, which helps Microsoft find and fix problems and improve our products and services, will not be sent back to Microsoft. ||[Learn how to turn off traffic to all of the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#bkmk-priv-feedback)|
+|||TLSv1.2/HTTPS/HTTP|v10.events.data.microsoft.com|
+|||TLSv1.2/HTTPS/HTTP|v20.events.data.microsoft.com|
+|||HTTP|www.microsoft.com|
+||The following endpoints are used by Windows Error Reporting. To turn off traffic for these endpoints, enable the following Group Policy: Administrative Templates > Windows Components > Windows Error Reporting > Disable Windows Error Reporting. This means error reporting information will not be sent back to Microsoft.|TLSv1.2|telecommand.telemetry.microsoft.com|
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-21H1-endpoints.md:73]:
```suggestion
||The following endpoints are used by Windows Error Reporting. To turn off traffic for these endpoints, enable the following Group Policy: **Administrative Templates** > **Windows Components** > **Windows Error Reporting** > **Disable Windows Error Reporting**. This means error reporting information will not be sent back to Microsoft.|TLSv1.2|telecommand.telemetry.microsoft.com|
```

File: windows/privacy/manage-windows-21H1-endpoints.md (hunk 9 above)
Comment (TomLayson) [reply windows/privacy/manage-windows-21H1-endpoints.md:35]: This doesn't seem right to me.  Per https://www.merriam-webster.com/dictionary/derive we should use derive as is or change it to "arrive at".  Thoughts?

File: windows/privacy/manage-windows-21H1-endpoints.md [hunk 14]
```diff
@@ -0,0 +1,162 @@
...
+|Microsoft Edge|||[Learn how to turn off traffic to all of the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#13-microsoft-edge)|
+||This traffic is related to the Microsoft Edge browser.|HTTPS|iecvlist.microsoft.com|
+||The following endpoint is used by Microsoft Edge Update service to check for new updates. If you disable this endpoint, Microsoft Edge won’t be able to check for and apply new edge updates.|TLSv1.2/HTTPS/HTTP|msedge.api.cdp.microsoft.com|
+|Microsoft forward link redirection service (FWLink)|The following endpoint is used by the Microsoft forward link redirection service (FWLink) to redirect permanent web links to their actual, sometimes transitory, URL. FWlinks are similar to URL shorteners, just longer. If you disable this endpoint, Windows Defender won't be able to update its malware definitions; links from Windows and other Microsoft products to the Web won't work; and PowerShell updateable Help won't update. To disable the traffic, instead disable the traffic that's getting forwarded.|HTTP|go.microsoft.com|
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-21H1-endpoints.md:86]:
```suggestion
|Microsoft forward link redirection service (FWLink)|The following endpoint is used by the Microsoft forward link redirection service (FWLink) to redirect permanent web links to their actual, sometimes transitory, URL. FWlinks are similar to URL shorteners, just longer. If you disable this endpoint, Windows Defender won't be able to update its malware definitions; links from Windows and other Microsoft products to the web won't work; and PowerShell updateable Help won't update. To disable the traffic, instead, disable the traffic that's getting forwarded.|HTTP|go.microsoft.com|
```

File: windows/privacy/manage-windows-21H1-endpoints.md [hunk 15]
```diff
@@ -0,0 +1,162 @@
...
+|||TLSv1.2/HTTPS/HTTP|settings-win.data.microsoft.com|
+|||HTTPS|settings.data.microsoft.com|
+|Skype|The following endpoint is used to retrieve Skype configuration values. To turn off traffic for this endpoint, either uninstall the app or disable the Microsoft Store. If you disable the Microsoft store, other Microsoft Store apps cannot be installed or updated. Additionally, the Microsoft Store won't be able to revoke malicious apps and users will still be able to open them.||[Learn how to turn off traffic to all of the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#bkmk-windowsstore)|
+|||HTTPS/HTTP|*.pipe.aria.microsoft.com|
+|||TLSv1.2/HTTPS/HTTP|config.edge.skype.com|
+|Teams|The following endpoint is used for Microsoft Teams application.||[Learn how to turn off traffic to all of the following endpoint(s).]( manage-connections-from-windows-operating-system-components-to-microsoft-services.md#26-microsoft-store)|
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-21H1-endpoints.md:115]:
```suggestion
|Teams|The following endpoint is used for Microsoft Teams application.||[Learn how to turn off traffic to all of the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#26-microsoft-store)|
```

File: windows/privacy/manage-windows-21H1-endpoints.md [hunk 16]
```diff
@@ -0,0 +1,162 @@
...
+|||TLSv1.2/HTTPS/HTTP|arc.msn.com|
+|||HTTPS|ris.api.iris.microsoft.com|
+|Windows Update|The following endpoint is used for Windows Update downloads of apps and OS updates, including HTTP downloads or HTTP downloads blended with peers. If you turn off traffic for this endpoint, Windows Update downloads will not be managed, as critical metadata that is used to make downloads more resilient is blocked. Downloads may be impacted by corruption (resulting in re-downloads of full files). Additionally, downloads of the same update by multiple devices on the same local network will not use peer devices for bandwidth reduction.||[Learn how to turn off traffic to all of the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#bkmk-updates)|
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-21H1-endpoints.md:124]:
```suggestion
|Windows Update|The following endpoint is used for Windows Update downloads of apps and operating system updates, including HTTP downloads or HTTP downloads blended with peers. If you turn off traffic for this endpoint, Windows Update downloads will not be managed, as critical metadata that is used to make downloads more resilient is blocked. Downloads may be impacted by corruption (resulting in re-downloads of full files). Additionally, downloads of the same update by multiple devices on the same local network will not use peer devices for bandwidth reduction.||[Learn how to turn off traffic to all of the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#bkmk-updates)|
```

File: windows/privacy/manage-windows-21H1-endpoints.md [hunk 17]
```diff
@@ -0,0 +1,162 @@
...
+|||TLSv1.2/HTTPS/HTTP|oneclient.sfx.ms|
+|||HTTPS| logincdn.msauth.net|
+|Settings|The following endpoint is used as a way for apps to dynamically update their configuration. Apps such as System Initiated User Feedback and the Xbox app use it. If you turn off traffic for this endpoint, an app that uses this endpoint may stop working.||[Learn how to turn off traffic to all of the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#bkmk-priv-feedback)|
+|||TLSv1.2/HTTPS/HTTP|settings-win.data.microsoft.com|
+|||HTTPS|settings.data.microsoft.com|
+|Skype|The following endpoint is used to retrieve Skype configuration values. To turn off traffic for this endpoint, either uninstall the app or disable the Microsoft Store. If you disable the Microsoft store, other Microsoft Store apps cannot be installed or updated. Additionally, the Microsoft Store won't be able to revoke malicious apps and users will still be able to open them.||[Learn how to turn off traffic to all of the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#bkmk-windowsstore)|
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-21H1-endpoints.md:112]:
```suggestion
|Skype|The following endpoint is used to retrieve Skype configuration values. To turn off traffic for this endpoint, either uninstall the app or disable the Microsoft Store. If you disable the Microsoft Store, other Microsoft Store apps cannot be installed or updated. Additionally, the Microsoft Store won't be able to revoke malicious apps and users will still be able to open them.||[Learn how to turn off traffic to all of the following endpoint(s).](manage-connections-from-windows-operating-system-components-to-microsoft-services.md#bkmk-windowsstore)|
```

File: windows/privacy/manage-windows-21H1-endpoints.md [hunk 18]
```diff
@@ -0,0 +1,162 @@
...
+|||TLSv1.2/HTTPS/HTTP|*.prod.do.dsp.mp.microsoft.com|
+|||HTTP|emdl.ws.microsoft.com|
+||The following endpoints are used to download operating system patches, updates, and apps from Microsoft Store. If you turn off traffic for these endpoints, the device will not be able to download updates for the operating system.|TLSv1.2/HTTPS/HTTP|*.dl.delivery.mp.microsoft.com|
+|||HTTP|*.windowsupdate.com|
+||The following endpoints enable connections to Windows Update, Microsoft Update, and the online services of the Store. If you turn off traffic for these endpoints, the device will not be able to connect to Windows Update and Microsoft Update to help keep the device secure. Also, the device will not be able to acquire and update apps from the Store. These are dependent on also enabling "Device authentication" and "Microsoft Account" endpoints.|TLSv1.2/HTTPS/HTTP|*.delivery.mp.microsoft.com|
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-21H1-endpoints.md:129]:
```suggestion
||The following endpoints enable connections to Windows Update, Microsoft Update, and the online services of the Store. If you turn off traffic for these endpoints, the device will not be able to connect to Windows Update and Microsoft Update to help keep the device secure. Also, the device will not be able to acquire and update apps from the Microsoft Store. These are dependent on also enabling "Device authentication" and "Microsoft Account" endpoints.|TLSv1.2/HTTPS/HTTP|*.delivery.mp.microsoft.com|
```

File: windows/privacy/manage-windows-21H1-endpoints.md [hunk 19]
```diff
@@ -0,0 +1,162 @@
...
+- [Windows 10, version 2004, connection endpoints for non-Enterprise editions](windows-endpoints-2004-non-enterprise-editions.md)
+- [Windows 10, version 1909, connection endpoints for non-Enterprise editions](windows-endpoints-1909-non-enterprise-editions.md)
+- [Windows 10, version 1903, connection endpoints for non-Enterprise editions](windows-endpoints-1903-non-enterprise-editions.md)
+- [Windows 10, version 1809, connection endpoints for non-Enterprise editions](windows-endpoints-1809-non-enterprise-editions.md)
+- [Windows 10, version 1803, connection endpoints for non-Enterprise editions](windows-endpoints-1803-non-enterprise-editions.md)
+- [Windows 10, version 1709, connection endpoints for non-Enterprise editions](windows-endpoints-1709-non-enterprise-editions.md)
+
+## Related links
+
+- [Office 365 URLs and IP address ranges](https://support.office.com/en-us/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2?ui=en-US&rs=en-US&ad=US)
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-windows-21H1-endpoints.md:161]:
```suggestion
- [Office 365 URLs and IP address ranges](https://support.office.com/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2?ui=en-US&rs=en-US&ad=US)
```

File: windows/privacy/manage-windows-21H1-endpoints.md (hunk 9 above)
Comment (JohanFreelancer9) [reply windows/privacy/manage-windows-21H1-endpoints.md:35]: Thanks, @TomLayson, I stand corrected. Of course, you are right. Please ignore my suggestion.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by JohanFreelancer9** [comment]
@TomLayson - thank you for committing my suggested changes, but you seem to have missed my final three suggestions. Could you kindly commit them as well if you agree with them? Many thanks.

**Comment by TomLayson** [comment]
Hi Johan,

I had 1 question about wording.  I have accepted the rest of the changes.  I'm new to GitHub, so if I need to do something, please let me know.

Thanks
Tom Layson
Privacy Program Manager 2
Windows, Browsers & Devices Privacy
https://aka.ms/wdgprivacy

General Privacy Questions? Check out our Stack Overflow page: https://aka.ms/wbdprivacyquestions
Request a privacy consult or review: https://aka.ms/wbdprivacyengage

**Comment by JohanFreelancer9** [comment]
> Hi Johan, I had 1 question about wording. I have accepted the rest of the changes. I'm new to GitHub, so if I need to do something, please let me know. Thanks Tom Layson Privacy Program Manager 2 Windows, Browsers & Devices Privacy https://aka.ms/wdgprivacy General Privacy Questions? Check out our Stack Overflow page: https://aka.ms/wbdprivacyquestions Request a privacy consult or review: https://aka.ms/wbdprivacyengage From: JohanFreelancer9 ***@***.***> Sent: Thursday, September 23, 2021 12:57 AM To: MicrosoftDocs/windows-itpro-docs ***@***.***> Cc: Tom Layson ***@***.***>; Mention ***@***.***> Subject: Re: [MicrosoftDocs/windows-itpro-docs] Rtb update (#9963) @TomLayson<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTomLayson&data=04%7C01%7Ctomlayson%40microsoft.com%7C86c269e0f41c4838aa6508d97e67ca05%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637679806622309384%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=f1N3BsccAYfcaqFGvDR%2BgyNSWHCGSw5FNX6cT23pSA4%3D&reserved=0> - thank you for committing my suggested changes, but you seem to have missed my final three suggestions. Could you kindly commit them as well if you agree with them? Many thanks. - You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fwindows-itpro-docs%2Fpull%2F9963%23issuecomment-925584686&data=04%7C01%7Ctomlayson%40microsoft.com%7C86c269e0f41c4838aa6508d97e67ca05%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637679806622309384%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=xEVqbrjmq3nm9ytDswm5yhpf3vsyYpxPJpb2%2BTMY9Go%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAT3S7MCFALPHTNB5ATZ7U7LUDLMWPANCNFSM5EJQTBAQ&data=04%7C01%7Ctomlayson%40microsoft.com%7C86c269e0f41c4838aa6508d97e67ca05%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637679806622319376%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=V7Yhht5AwgXZUgJuJPkijaaBXWbkdT4sRFmzQ9XZ2kw%3D&reserved=0>. Triage notifications on the go with GitHub Mobile for iOS<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Ctomlayson%40microsoft.com%7C86c269e0f41c4838aa6508d97e67ca05%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637679806622329374%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=8408YpmcyaedzpJAVbW2H8ijRQdRNPyMzHTr5o4xGqM%3D&reserved=0> or Android<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Ctomlayson%40microsoft.com%7C86c269e0f41c4838aa6508d97e67ca05%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637679806622329374%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=%2BncCcEaTxXX9RW4YA%2FY00898ooinqTY15j%2BIupse5iU%3D&reserved=0>.

Hi Tom,

Thank you for committing all the changes. It is all good. You can please just mark your question as "Resolved". Thanks again.

Johan

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by TomLayson** [comment]
@JohanFreelancer9  I don't see an option to do anything with the final comment.  The active option on the page is "Close with comment".

**Comment by JohanFreelancer9** [comment]
@TomLayson - If you select "**...** at the top of your comment, you should get the option to resolve or delete. Could you please have a look?

**Comment by aczechowski** [comment]
Apologies for the delay in finalizing this PR, but thank you @TomLayson for the contribution. I fixed some conflicts due to the time lapse, and will merge this now. The changes will publish live around 11 AM Pacific on 4 May.

**Pull request merged**